.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on: "push"
+
+concurrency:
+   group: ci-${{ github.head_ref || github.ref }}
+   cancel-in-progress: true
+
+jobs:
+  tests:
+    name: "Node ${{ matrix.node-version }} - ${{ matrix.os }} "
+    runs-on: "${{ matrix.os }}-latest"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
Can we keep `[6.x, 8.x, 10.x]` for this PR? And drop/add support for Node versions in a separate one?